src/olympia/editors/models.py

```diff
@@ -243,6 +243,52 @@ def base_query(self):
         return q
 
 
+class ViewAllList(ViewQueue):
+    guid = models.CharField(max_length=255)
+    version_date = models.DateTimeField()
+    _author_ids = models.CharField(max_length=255)
+    _author_usernames = models.CharField()
+    review_date = models.DateField()
+    review_version_num = models.CharField(max_length=255)
+
+    @property
+    def authors(self):
+        ids = self._explode_concat(self._author_ids)
+        usernames = self._explode_concat(self._author_usernames, cast=unicode)
+        return zip(ids, usernames)
+
+    def base_query(self):
+        q = super(ViewAllList, self).base_query()
+        q['select'].update({
+            'guid': 'addons.guid',
+            '_author_ids': """GROUP_CONCAT(authors.user_id)""",
+            '_author_usernames': """GROUP_CONCAT(users.username)""",
+            'version_date': 'versions.nomination',
+            'review_date': 'rversions.reviewed',
+            'review_version_num': 'rversions.version',
+        })
+        q['from'].extend([
+            """LEFT JOIN addons_users AS authors
+                    ON addons.id = authors.addon_id""",
+            'LEFT JOIN users as users ON users.id = authors.user_id',
+            """LEFT JOIN (
+                SELECT id, addon_id, reviewed, version FROM versions
+                WHERE reviewed IS NOT NULL
+                ORDER BY id desc
+                ) AS rversions
```

Comment:
I find `rversion` cryptic. I initially read it as a typo of `revisions`. If `reviewed_versions` fits that would be better.